requestor-tutorials/flash-tutorial-of-requestor-development.md

```diff
@@ -86,7 +86,7 @@ git --version
 #### No crypto assets needed \(for now!\)
 
 {% hint style="info" %}
-Alpha lives on the Rinkeby Testnet. You don't need any real ETH or GLM tokens to start this tutorial. You also don't need to do anything to get test tokens! These test assets are acquired by the daemon in one of the steps below.
+Alpha lives on the Rinkeby Testnet. You don't need any real ETH or tGLM tokens to start this tutorial. You also don't need to do anything to get test tokens! These test assets are acquired by the daemon in one of the steps below.
```

Comment:
this is a tricky one, it says `You don't need real...` so GLM should be the name there..

would add the rETH and tGLM names later in the sentence:

```suggestion
Alpha lives on the Rinkeby Testnet. You don't need any real ETH or GLM tokens to start this tutorial. You also don't need to do anything to get test tokens! These test assets, rETH and tGLM, are acquired by the daemon in one of the steps below.
```